migrations/6_deploy_bridged_token_owl_on_xdai.js

```diff
@@ -0,0 +1,18 @@
+/* global artifacts, web3 */
+
+const BridgedTokenOWL = artifacts.require('BridgedTokenOWL')
+
+module.exports = async function (deployer, network, accounts) {
+  if (process.env.DEPLOY_OWL_ONLY) {
```

Comment:
Maybe this deployment script should only deploy if `networkId === 100`, or else couldn't `BridgedTokenOWL` potentially get accidentally deployed on Mainnet as well?